pyigm/cgm/cgmsurvey.py

```diff
@@ -190,6 +326,97 @@ def to_json_tarball(self, outfil):
             os.remove(jfile)
         os.rmdir(tmpdir)
 
+    def load_json(self, jfile, build_data=True, build_sys=False, verbose=True, **kwargs):
+        """
+        Parameters
+        ----------
+        jfile
+        build_data
+        build_sys
+        kwargs
+
+        Returns
+        -------
+
+        """
+        # Load
+        self._dict = ltu.loadjson(jfile)
+        # Generate
+        if build_sys:
+            self.build_systems_from_dict(**kwargs)
+
+        # Galaxy coords
+        ras = [self._dict[key]['RA'] for key in self._dict.keys()]
+        decs = [self._dict[key]['DEC'] for key in self._dict.keys()]
+        self.coords = SkyCoord(ra=ras, dec=decs, unit='deg')
+
+        # Sightline coords
+        ras = [self._dict[key]['igm_sys']['RA'] for key in self._dict.keys()]
+        decs = [self._dict[key]['igm_sys']['DEC'] for key in self._dict.keys()]
+        self.scoords = SkyCoord(ra=ras, dec=decs, unit='deg')
+
+        # Data table
+        if build_data:
+            self.data_from_dict()
+        # Return
+        return
+
+    def load_tarball(self, tfile, build_data=True, build_sys=False, llist=None, verbose=True, **kwargs):
+        """
+        Parameters
+        ----------
+        tfile
+        build_data
```

Comment:
Some docstring could help for at least 'build_data', whose usage may not be clear to casual users.

Reply:
added